redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/DatabaseV4Endpoint.java

```diff
@@ -5,9 +5,9 @@
 import java.util.Set;
 
 import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
```

Comment:
The CloudBreak codebase is inconsistent in its choice of which non-null annotation to use. I prefer javax.annotation.Nonnull.